Makefile

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+SHELL=/bin/bash
+TAG := latest
+
+.DEFAULT_GOAL := help
+.PHONY: help
+help: ## Print this help
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: test unit-test e2e-test build build-dev push
+
+unit-test: ## Run unit tests for all functions
```

Comment:
A possible solution to the comment above:

This makefile should optionally take a language parameter (Go or TS):

// Test both Go and TS
make unit-test 

// Test Go. This target can be called from workflow.
make unit-test-go

// Test TS. This target can be called from workflow.
make unit-test-ts